x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/data_input_flyout/steps/rules/sub_steps/copy_export_query/index.tsx

```diff
@@ -5,22 +5,35 @@
  * 2.0.
  */
 
-import React from 'react';
+import React, { useMemo } from 'react';
 import type { EuiStepProps, EuiStepStatus } from '@elastic/eui';
-import { CopyExportQuery } from './copy_export_query';
+import { CopyExportedSplunkQuery } from './copy_exported_splunk_query';
 import * as i18n from './translations';
+import { CopyExportedQradarQuery } from './copy_exported_qradar_query';
+import { MigrationSource } from '../../../../../../../common/types';
 
 export interface CopyExportQueryStepProps {
   status: EuiStepStatus;
+  migrationSource: MigrationSource;
   onCopied: () => void;
 }
+
 export const useCopyExportQueryStep = ({
+  migrationSource,
   status,
   onCopied,
 }: CopyExportQueryStepProps): EuiStepProps => {
+  const COPY_EXPORT_QUERY_SUPPORTED_SOURCES: Record<MigrationSource, React.ReactNode> = useMemo(
+    () => ({
+      [MigrationSource.SPLUNK]: <CopyExportedSplunkQuery onCopied={onCopied} />,
+      [MigrationSource.QRADAR]: <CopyExportedQradarQuery />,
+    }),
+    [onCopied]
+  );
+
   return {
     title: i18n.RULES_DATA_INPUT_COPY_TITLE,
```

Comment:
This need to be changed. Currently it says `Copy Rules Query`, which is incorrect in the design. It should say `Export rules` which matches our instructions